src/components/SignInForm/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Link from 'next/link'
+import Form from './Form'
+
+const SignInForm = () => {
+  return (
+    <section className="sign-in-form">
+      <header className="sign-in-form__header">Sign In</header>
+      <div className="sign-in-form__body">
+        <Form />
+      </div>
+      <footer className="sign-in-form__footer">
+        Don't have an account?
+        <Link href="/sign-up">
+          <a className="sign-in-form__link">Create here</a>
```

Comment:
I think for all the sentence from ''Don't have an account? Create here'' is better than ''Don't have an account? Create account'' via redundant words in the sentence.

Reply:
Let's see these examples >> [https://uxdesign.cc/designing-a-user-friendly-login-25855ae0cc88](). And I've never seen 'Create here' before. Do you have any references to share, in case I miss something 👀 ?

Reply:
Good reference, so I'll change into 'create an account' 😁